kernel/main.cpp

```diff
@@ -77,39 +78,27 @@ void kernel_main(void *boot_info, uint32_t magic) {
     // Print some info to show we did things right
     rtc_print();
     // Get the CPU vendor and model data to print
-    char *vendor = (char *)cpu_get_vendor();
-    char *model = (char *)cpu_get_model();
+    const char *vendor = cpu_get_vendor();
+    const char *model = cpu_get_model();
     kprintf(DBG_INFO "%s\n", vendor);
     kprintf(DBG_INFO "%s\n", model);
-    // Start the serial debugger
-    kprintf(DBG_INFO "Starting serial debugger...\n");
     // Print out the CPU vendor info
     rs232_print(vendor);
     rs232_print("\n");
     rs232_print(model);
     rs232_print("\n");
 
-    // Done
-    kprintf(DBG_OKAY "Done.\n");
-
     tasks_init();
     task_t compute, status;
-    tasks_new(find_primes, &compute, TASK_READY, "prime_compute");
-    tasks_new(show_primes, &status, TASK_READY, "prime_display");
+    tasks_new(task_find_primes, &compute, TASK_READY, "prime_compute");
+    tasks_new(task_show_primes, &status, TASK_READY, "prime_display");
+    //tasks_new(task_spinner, &status, TASK_READY, "spinner");
```

Comment:
Make sure you create a new `task_t` for the spinner.

Reply:
That fixed the printing issue. Good catch.

Reply:
Resolved in b246794.